### PR TITLE
Impl 301 1

### PR DESCRIPTION
**Pull request by Huyeng**
Enhance mapping function to be able to present the selected layers of map on profile page.
see this: 
https://github.com/OpenDevelopmentMekong/wp-odm_profile_pages/pull/60

### CODE REVIEW

File: inc/layers.php [hunk 1]
```diff
@@ -17,16 +17,6 @@ function add_layers_box_to_post_types(){
       return $supported_post_types;
     }
 
-    function get_localization_language($site=""){
```
Comment (acorbi) [inline inc/layers.php]: @Huyeng this function is unnecessary, we can re-use odm_language_manager()->get_the_language_by_site()

File: inc/layers.php [hunk 2]
```diff
@@ -817,6 +806,10 @@ function get_layer($post_id = false) {
             //, 'legend' => $layer_legend
         );
 
+        if (!empty($in_category)):
```
Comment (acorbi) [inline inc/layers.php]: @Huyeng this solves an error which appears when there is no category on the wordpress instance (as it was the case on my local)

File: inc/layers.php [hunk 3]
```diff
@@ -808,7 +798,6 @@ function get_layer($post_id = false) {
             'title' => get_the_title(),
             //'post_content' => $content, //content(999)
```
Comment (acorbi) [inline inc/layers.php]: @Huyeng can we remove all these comments?

File: inc/layers.php (hunk 1 above)
Comment (Huyeng) [inline inc/layers.php]: Of course.  It should be updated long time a go.

File: inc/layers.php (hunk 3 above)
Comment (Huyeng) [inline inc/layers.php]: Yes, we can clear them all. Would you like me to do it?

**Pull request merged**